WEB/Src/DependencyCollector/DependencyCollector/Implementation/RemoteDependencyConstants.cs

```diff
@@ -10,6 +10,7 @@ internal static class RemoteDependencyConstants
         public const string AzureTable = "Azure table";
         public const string AzureQueue = "Azure queue";
         public const string AzureDocumentDb = "Azure DocumentDB";
+        public const string AzureCosmosDb = "Azure CosmosDB";
```

Comment:
DocumentDb is a legacy name and it'd be great to use CosmosDb instead.

TODO: need to run it by Portal team as well.

Reply:
Checked with @OsvaldoRosado - we should populate resource provider namespace - `Microsoft.DocumentDB`